engine/runtime/src/main/java/org/enso/interpreter/node/callable/InvokeMethodNode.java

```diff
@@ -620,9 +620,12 @@ Object doFallback(
       @CachedLibrary(limit = "10") TypesLibrary methods,
       @CachedLibrary(limit = "10") InteropLibrary interop,
       @CachedLibrary(limit = "10") WarningsLibrary warnings,
-      @Cached MethodResolverNode anyResolverNode) {
+      @Cached MethodResolverNode resolverNode) {
     var ctx = EnsoContext.get(this);
-    Function function = anyResolverNode.expectNonNull(self, ctx.getBuiltins().any(), symbol);
+    Function function = resolverNode.execute(ctx.getBuiltins().function(), symbol);
+    if (function == null) {
```

Comment:
This condition appears to be spurious. `Any` is already a supertype of `Function` so the second check will not resolve it if the first one didn't.

Reply:
Thanks. Let's try: 5ffa216 version.